src/main/resources/static/pages/ticket-detail.html

```
@@ -59,7 +59,8 @@ <h2>Ladda upp dokument</h2>
 </main>
 
 <script src="/js/common.js"></script>
-<script src="/js/ticket-detail.js"></script>
+<script src="/js/online-staff.js"></script>
+<script src="/js/ticket-detail.js"></script>>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)